sdk/mediaservices/ci.yml

```diff
@@ -29,3 +29,5 @@ extends:
     Artifacts:
       - name: Microsoft.Azure.Media.LiveVideoAnalytics.Edge
         safeName: MicrosoftAzureMediaLiveVideoAnalyticsEdge
```

Comment:
Can we remove these?

Reply:
not yet, just in case if we need any bugfixes on the old sdk.